src/System.ComponentModel.TypeConverter/src/System/ComponentModel/EnumConverter.cs

```diff
@@ -134,6 +135,39 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
                 return Enum.Format(EnumType, value, "G");
             }
 
+            if (destinationType == typeof(InstanceDescriptor) && value != null)
+            {
+                string enumName = ConvertToInvariantString(context, value);
+
+                if (EnumType.IsDefined(typeof(FlagsAttribute), false) && enumName.IndexOf(',') != -1)
```

Comment:
Nit: enumName.IndexOf could be changed to enumName.Contains

Reply:
Sure I can make this fix over the desktop code.